web/sites/default/config/swiftype_integration.settings.yml

```diff
@@ -0,0 +1,7 @@
+swiftype_integration_install_key: K2y8KFxi3pKyFfU1-csZ
```

Comment:
@RichardDavies, is there any concern with these keys being published publicly? Are they secrets or just configuration information?